app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -19,14 +19,46 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.10.3\n"
 
-#: app/create_app.py:89
+#: app/create_app.py:91
 msgid "Apply for funding to save an asset in your community"
 msgstr "Gwneud cais am gyllid i achub ased yn eich cymuned"
 
 #: app/filters.py:14 app/filters.py:25
 msgid "at"
 msgstr "am"
 
+#: app/models/language.py:5
+msgid "English"
+msgstr "Saesneg"
+
+#: app/models/language.py:6
+msgid "Welsh"
+msgstr "Cymraeg"
+
+#: app/models/statuses.py:9
+msgid "Not Started"
+msgstr "Rydych wedi dechrau"
+
+#: app/models/statuses.py:10
+msgid "In Progress"
+msgstr "In Progress Welsh"
+
+#: app/models/statuses.py:11
+msgid "Assessment complete"
+msgstr "o'r meini prawf asesu"
```

Comment:
Is this translation correct? If it is then should we have a capital at the beginning?

```suggestion
msgstr "O'r meini prawf asesu"
```